uiowa/uiowa
# Feature: Admissions pdf fixes

This pull requests fixes a few missed items missed in the pdf-styles pr.  

# How to test

- `blt ds --site=admissions.uiowa.edu`
- `yarn workspace admissions_core build`
- `drush @admissions.local cr`
- Go to http://admissions.local.drupal.uiowa.edu/areas-of-study/print-links
- Verify that the pdf looks similar to the example below

<img width="997" alt="Screen Shot 2021-02-22 at 10 18 27 AM" src="https://user-images.githubusercontent.com/1036433/108736766-88744f00-74f7-11eb-88f0-7aac84f1f292.png">
